### PR TITLE
added extra legend options as argument

### PR DESCRIPTION
**Pull request by eliasm**

### CODE REVIEW

**Comment by egeerardyn** [comment]
@matlab2tikzbot: test this PR

**Comment by okomarov** [comment]
@eliasm It would be great if you had a test figure with two subplots with each using a different property :)

**Comment by egeerardyn** [comment]
@eliasm Could you please elaborate a bit why you want to add this?
If I'm not mistaken, you can get exactly the same result by setting
```latex
\pgfplotsset{every axis legend/.append style={...}}
```
either to your LaTeX preamble or by passing something equivalent to the options we already have (e.g. `extraTikzpictureOptions`).

Since the code is clean, I don't mind adding it to the repository, but I hope you understand it will be deprecated when #393 is implemented (which should be in a few months).

File: src/matlab2tikz.m [hunk 1]
```diff
@@ -218,6 +222,7 @@ function matlab2tikz(varargin)
 ipp = ipp.addParamValue(ipp, 'extraCodeAtEnd', {}, @isCellOrChar);
 ipp = ipp.addParamValue(ipp, 'extraAxisOptions', {}, @isCellOrChar);
 ipp = ipp.addParamValue(ipp, 'extraTikzpictureOptions', {}, @isCellOrChar);
+ipp = ipp.addParamValue(ipp, 'extraTikzLegendOptions', {}, @isCellOrChar);
```
Comment (miscco) [inline src/matlab2tikz.m]: While you at it, can you change the above to extraTikzPictureOptions, to stay consistend. I assume previously tikzpicture was considered one word.

**Comment by eliasm** [comment]
@egeerardyn You are right, the exact same effect can be achieved by:
```
matlab2tikz('plot.tex', 'extraAxisOptions', 'every axis legend/.append style={transpose legend, legend columns=3}');
```
Thanks for looking at it!

**Pull request closed without merging**